examples/ota-provider-app/ota-provider-common/OTAProviderExample.h

```diff
@@ -42,7 +42,16 @@ class OTAProviderExample : public OTAProviderDelegate
                                            uint32_t newVersion) override;
     EmberAfStatus HandleNotifyUpdateApplied(const chip::ByteSpan & updateToken, uint32_t currentVersion) override;
 
+    enum queryImageBehaviorType {
```

Comment:
Is this enum necessary? Why can't you just reuse `EMBER_ZCL_OTA_QUERY_STATUS_NOT_AVAILABLE`, etc?

Reply:
queryImageBehaviorType represents an internal state which does not strictly translate into an QueryImage status value. For example, in the kRespondWithUpdateAvailable state we may still send out EMBER_ZCL_OTA_QUERY_STATUS_NOT_AVAILABLE. Having a single enum denote both items would be misleading.

Reply:
Got it - thanks